### ISSUE TITLE
Window control buttons looks bad In Win10 then title bar is hidden

### ISSUE THREAD

**Issue by Cepr0** [Croydon/vertical-tabs-reloaded#64]
![2016-09-01_20-30-17](https://cloud.githubusercontent.com/assets/10778600/18177693/0fabc3d6-7084-11e6-8aaf-a4e6363cd203.gif)

Add-on version: 0.7.0
Firefox version: 48.0.2
OS + OS-Version: Win10
Other tab + user interface related add-ons:  TabMix Plus

**Comment by myBestSoftAndPref** [comment]
It happens because title bar is not hidden. It's just partially invisible.  
Hot fix for Stylish addon or userChrome.css:
```css
@namespace url(http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul);

#titlebar
{
  display : none !important;
}
```

**Comment by myBestSoftAndPref** [comment]
Or you can use https://addons.mozilla.org/en-US/firefox/addon/hide-caption-titlebar-plus-sma

**Comment by Cepr0** [comment]
Thanks! I will try it..

**Comment by myBestSoftAndPref** [comment]
There is same bug under Windows 7 with Classic Theme at least.

**Comment by Croydon** [comment]
You can toggle the titlebar in the add-on settings. It takes more space, but shouldn't overlap then.
Completely hiding the control elements doesn't sound like a solution suitable for most users.
What could I do here to make your user case better?

**Comment by myBestSoftAndPref** [comment]
> What could I do here to make your user case better?

1. You can copy solution from https://addons.mozilla.org/en-US/firefox/addon/hide-caption-titlebar-plus-sma
2. You can create and add three simple buttons on tool bar which call three function: close, minimize/maximize, inwindow/fullsize. I mean it can be just simple button like other addon's buttons. Or it can be group of button.

**Comment by anonymous13587** [comment]
Why not place the navigation bar content and window controls in the same toolbar? The unoccupied space could be draggable with some reserved next to the window controls like the default tab bar. You could also add some space above the bar when the window is not maximized, as with the default theme.
![capture](https://cloud.githubusercontent.com/assets/25938118/23183580/8bdf1cd4-f84a-11e6-8cbc-759a1453e5b2.PNG)
![capture2](https://cloud.githubusercontent.com/assets/25938118/23183581/8be33d46-f84a-11e6-8435-d553c557e1f8.PNG)

**Comment by Croydon** [comment]
@anonymous13587 I agree that his is one of the best designs, but I won't be able to do anything like that with WebExtensions.

The Firefox team wants to support vertical tabs add-on in some way, including hiding the default horizontal tabbrowser, so they have to come up with some solution for this.

**Comment by Croydon** [comment]
This will be fixed with #128.

**Comment by Croydon** [comment]
To be more exact: Maybe not, but there is nothing I can really do here.

The WebExtension version won't hide the tab titlebar via default, now will it be possible to toggle it from within the VTR options anymore. Together with hiding the default tab browser, this really becomes a bug, which only Firefox itself can solve.
